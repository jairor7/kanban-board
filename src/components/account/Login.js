import React from "react";
import { connect } from "react-redux";
import { login } from "../../store/redux/actions/loginAction";

export const Login = ({ login }) => {
  return (
    <div>
      <button onClick={login}>Login</button>
    </div>
  );
};
const mapStateToProps = (state) => ({});

const mapDispatchToProps = (dispatch) => ({
  login: () => dispatch(login()),
});

export default connect(mapStateToProps, mapDispatchToProps)(Login);
