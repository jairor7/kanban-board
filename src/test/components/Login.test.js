import Login from "../../components/account/Login";
import React from "react";
import toJSON from "enzyme-to-json";
import { shallow } from "enzyme";

test("should render Login correctly", () => {
  const wrapper = shallow(<Login />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
