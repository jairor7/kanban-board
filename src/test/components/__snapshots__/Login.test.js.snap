// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Login correctly 1`] = `
<div>
  Login
</div>
`;
